tile/lang/compose.cc

```diff
@@ -2,6 +2,7 @@
 
 #include <algorithm>
 #include <sstream>
+#include <stack>
```

Comment:
note (to Tim and just general FYI): this is a change that was requested by cpplint.